dashboard/app/models/course.rb

```diff
@@ -55,6 +56,14 @@ def localized_version_title
     I18n.t("data.course.name.#{name}.version_title", default: version_year)
   end
 
+  # Any course with a plc_course or no family_name is considered stable.
+  # All other courses must specify an is_stable boolean property.
+  def stable_or_default
+    return true if plc_course || !family_name
```

Comment:
should we consider `version_year` here at all, or is there an assumption that `family_name ⇒ version_year`?

Reply:
great question. so far from what i understand, we've been going on that assumption, but @davidsbailey might have more insight here

Reply:
depending on family_name seems fine for now. currently, the presence of family_name implies the presence of version year in the two courses that have family names, but there's nothing programmatically enforcing this. I'd say, go ahead with this for now and maybe later we can add some checks to enforce any assumptions.